README.md

```diff
@@ -129,8 +129,8 @@ response.reload_until_finished! tries: 300 # default is 600
 ```
 
 In general, you use hash accessor syntax to query any direct attribute from
-the [response](http://transloadit.com/docs/assemblies#response-format).
-Methods suffixed by a question mark provide a more readable way of quering
+the [response](https://transloadit.com/docs/#assembly-status-response).
```

Comment:
You've changed this link and it now goes to the doc overview and not to any anchor. Is this what you want?

(Also is true for all the other links you've changed in here)